### PR TITLE
feat: Created Cart page

### PR DESCRIPTION
**Pull request by arjunbharti**
Created page for fakeSea:

- Increment or decrement product quantity
- Billing amount
- Place order action

Preview: https://deploy-preview-4--fakesea.netlify.app/pages/cart.html

### CODE REVIEW

**Review by SHIKHASINGH1506 — APPROVED** [review]
Added a suggestion regarding making flex as a utility class as you're using using it frequently. Rest are cool.

**Review by sahiba0915 — COMMENTED** [review]
looks good to me

File: styles/cart.css [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    width: 100vw;
+    justify-content: space-around;
+}
+
+.price-details{
+    padding: 2rem;
+    display: flex;
+    flex-direction: column;
+    width: 50%;
+    border: 2px solid var(--background);
```
Comment (xii-vi) [inline styles/cart.css:19]: try to give variable more generic name

**Review by xii-vi — APPROVED** [review]
_A tiny suggestion. Rest is okay to merge_

File: styles/cart.css [hunk 2]
```diff
@@ -0,0 +1,89 @@
+.cart-title{
+    font-weight: 600;
+    text-align: center;
+    margin-top: 2rem;
+}
+
+.cart-container{
+    display: flex;
```
[the remark this answers is not in the document]
Comment (arjunbharti) [reply styles/cart.css:8]: sure, will update that

File: styles/cart.css (hunk 1 above)
Comment (arjunbharti) [reply styles/cart.css:19]: Will take care of it. Thanks

**Pull request merged**